yb-voyager/src/callhome/diagnostics.go

```diff
@@ -264,6 +267,25 @@ type ImportDataPhasePayload struct {
 	ControlPlaneType    string `json:"control_plane_type"`
 }
 
+type YBClusterMetrics struct {
+	Timestamp                 time.Time    `json:"timestamp"`                      // time when the metrics were collected
+	CPUAvgPct                 float64      `json:"cpu_avg_pct"`                    // mean of node CPU% across all nodes
+	TserverAvgMemSoftLimitPct float64      `json:"tserver_avg_mem_soft_limit_pct"` // mean of node tserver-soft-limit% across all nodes
```

Comment:
nit: technically, it's the average of tserver's memory consumption % (as per the soft limit), 

So, this is verbose, but maybe `tserverAvgMemConsumptionPctSoftLimit / tserver_avg_mem_consumption_pct_soft_limit` ?

Reply:
removed it